plugins/inputs/openldap/openldap.go

```diff
@@ -0,0 +1,148 @@
+package openldap
+
+import (
+	"crypto/tls"
+	"fmt"
+	"gopkg.in/ldap.v2"
+	"strconv"
+	"strings"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/plugins/inputs"
+)
+
+type Openldap struct {
+	Host          string
+	Port          int
+	Tls           bool
+	TlsSkipverify bool
+	BindDn        string
+	BindPassword  string
+}
```

Comment:
Also, what about providing a path to a custom CA cert, so that users running their own cert aren't forced to use tls_skipverify?